Automattic/wp-calypso
# Update/48029 rework dashboard for wc move

#### Changes proposed in this Pull Request

* Fixes #48029  and #48026
* Replaces `Dashboard` with `StoreMoveNoticeView` component when `woocommerce/store-remove` flag is set to true
* Adds `StoreMoveNoticeView` component at the top of `Dashboard` when `woocommerce/store-deprecated` flag is set to true

**The text copy might change as it is not finalized yet.**


#### Screenshots
**woocommerce/store-deprecated**
![Screen Shot 2020-12-15 at 5 53 25 PM](https://user-images.githubusercontent.com/4723145/102294643-9eb1cd80-3efe-11eb-9ab4-2adc13a726ef.jpg)

**woocommerce/store-removed**
![Screen Shot 2020-12-15 at 5 53 37 PM](https://user-images.githubusercontent.com/4723145/102294651-a40f1800-3efe-11eb-8f31-62c675fd547d.jpg)


#### Testing instructions

1. Start Calypso locally
2. Navigate to http://calypso.localhost:3000/ 
3. Click "Switch Site" from the sidebar and choose your Business site with Store setup.
4. Replace "home" to "store" and append `?flags=woocommerce/store-removed` from your browser's URL and hit enter.
5. append `?flags=woocommerce/store-deprecated` to the URL and navigate to the dashboard. You should see the notice with "Store is moving to WooCommerce"
6.append `?flags=woocommerce/store-removed` to the URL and navigate to the dashboard. You should see the notice with "Store has moved to WooCommerce"



**URL replacement example:**

http://calypso.localhost:3000/home/calypsotest363078866.wpcomstaging.com

**to**

http://calypso.localhost:3000/store/calypsotest363078866.wpcomstaging.com?flags=store-removed
